<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>timerWidget</class>
 <widget class="QWidget" name="timerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>58</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="addLabel">
       <property name="text">
        <string>Add:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="noteEdit">
       <property name="text">
        <string>Notes...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="setTimeLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>At:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="setTimeBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="minsLabel">
       <property name="text">
        <string>mins</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="timeLCD">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>152</width>
         <height>38</height>
        </size>
       </property>
       <property name="digitCount">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string/>
       </property>
       <property name="toolTip">
        <string>Stop</string>
       </property>
       <property name="icon">
        <iconset resource="../brewken.qrc">
         <normaloff>:/images/alarm_stop.png</normaloff>:/images/alarm_stop.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setSoundButton">
       <property name="text">
        <string/>
       </property>
       <property name="toolTip">
        <string>Set Sound</string>
       </property>
       <property name="icon">
        <iconset resource="../brewken.qrc">
         <normaloff>:/images/sound.png</normaloff>:/images/sound.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string/>
       </property>
       <property name="toolTip">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../brewken.qrc">
         <normaloff>:/images/exit.svg</normaloff>:/images/exit.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playButton">
       <property name="text">
        <string/>
       </property>
       <property name="toolTip">
        <string>Play Alarm Sound</string>
       </property>
       <property name="icon">
        <iconset resource="../brewken.qrc">
         <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../brewken.qrc"/>
 </resources>
 <connections/>
</ui>
